frontend/src/utils.ts

```diff
@@ -23,11 +23,54 @@ export function userProfileUrl(username: string): string {
    return `https://github.com/${safeUsername}`;
 }
 
-export const dummyPulls: PullData[] = (process.env.DUMMY_PULLS || []) as PullData[];
+export const dummyPulls: PullData[] = fakeCIStatuses((process.env.DUMMY_PULLS || []) as PullData[]);
 export function hasDummyPulls() {
    return dummyPulls.length > 0;
 }
 
 export function newTab(url: string) {
    window.open(url, "_blank");
 }
+
+function fakeCIStatuses(pulls: PullData[]) {
+   const fakeContexts = ['psalm', 'deploy', 'unit-tests', 'api', 'integration', 'browser', 'bundle-analysis'];
+   pulls.forEach((pull) => {
+      pull.status.commit_statuses = fakeContexts.map((context) => {
```

Comment:
This looks like it's mutating the original objects in `pulls` (which might be confusing down the road, because we're also returning the mutated objects). If we don't want to mutate, we could consider using something like this:
```ts
return pulls.map((pull) => {
   const commit_statuses = fakeContexts.map((context) => {
       const now = unix();
       const startedAt = now - (Math.random() * 120 * 60);
       const duration = Math.random() * 10 * 60;
       const completedAt = startedAt + duration;
       return {data: {
          sha: "fake sha",
          target_url: "https://example.com",
          description: context,
          context: context,
          state: completedAt > now ? StatusState.pending : finishedState(),
          started_at: startedAt,
          completed_at: completedAt > now ? null : completedAt,
       }};
   });
   return {
      ...pull,
      status: {
         ...pull.status,
         commit_statuses,
      },
   }
```

Reply:
This is only used on an array of dummy pulls and that's only used in development, I'm OK with mutating.